tests/instrumentation/asyncio/aiohttp_client_tests.py

```diff
@@ -29,6 +29,7 @@
 #  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 import pytest  # isort:skip
+import yarl
```

Comment:
@alxpy we need to move this import below the `pytest.importorskip` line. `yarl` is only available if aiohttp is installed. This is a side effect of how our test matrix build works. It only installs the requirements needed for a certain test module, and we use `importskip` to gate that.

Only noticed this after submitting the review, sorry!

Reply:
no problem, done)